en/mkdocs.yml

```diff
@@ -228,6 +228,7 @@ nav:
                         - Third Party Key Manager: deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/third-party-key-managers.md
                     - As a Standalone Gateway: deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md
                     - Subscription Validation: deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/subscription-validation.md
+                    - Configure Multiple Gateways with Choreo Connect: deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md
```

Comment:
We should not list in the Concept menu.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/5161/commits/20c93edf8d58d0f82a02a3b4204f3a64954d3550